ToDoList/Controllers/HomeController.cs

```diff
@@ -104,6 +104,25 @@ public async Task<IActionResult> DeleteToDoListConfirmed(int id)
             return RedirectToAction(nameof(Index));
         }
 
+        [HttpGet]
+        public async Task<IActionResult> EditToDoList(int id)
+        {
+            TDList list = await TDListDb.GetToDoListById(_context, id);
+            if (list == null)
+            {
+                return NotFound();
+            }
+            return View(list);
+        }
+
+        [HttpPost]
+        public async Task<IActionResult> EditToDoList(TDList list)
+        {
+            await TDListDb.UpdateToDoList(_context, list);
+            TempData["Message"] = $"Name changed successfully";
```

Comment:
For the TempData["Message"], did you mean to add {list.Title} since it has the $ symbol?

Reply:
I originally planned to use the Title property, but i figured since the user is changing it, it'd be better to make this part more generic. I just forgot to take out the $ symbol.